spec.html

```diff
@@ -32,8 +32,118 @@ <h1>Well-known intrinsic objects</h1>
+		</td>
+		<td>
+			The wrapped function object.
+		</td>
+		</tr>
+		<tr>
+		<td>
+			[[Realm]]
+		</td>
+		<td>
+			Realm Record
+		</td>
+		<td>
+			The realm in which the wrapped function object was created.
+		</td>
+		</tr>
+		</tbody>
+	</table>
+	</emu-table>
+
+	<emu-clause id="sec-wrapped-function-exotic-objects-call-thisargument-argumentslist">
+	<h1>[[Call]] ( _thisArgument_, _argumentsList_ )</h1>
+	<p>The [[Call]] internal method of a wrapped function exotic object _F_ takes arguments _thisArgument_ (an ECMAScript language value) and _argumentsList_ (a List of ECMAScript language values). It performs the following steps when called:</p>
+	<emu-alg>
+		1. Let _target_ be _F_.[[WrappedTargetFunction]].
+		1. Assert: IsCallable(_target_) is *true*.
+		1. Let _targetRealm_ be ? GetFunctionRealm(_target_).
+		1. Let _callerRealm_ be ? GetFunctionRealm(_F_).
+		1. For each element _key_ of _argumentsList_, do
+			1. Let o be _argumentsList_[_key_].
+			1. Set _argumentsList_[_key_] to ? GetWrappedValue(_targetRealm_, _o_).
+		1. Let _wrappedThisArgument_ to ? GetWrappedValue(_targetRealm_, _thisArgument_).
+		1. Let _value_ be ? Call(_target_, _wrappedThisArgument_, _argumentsList_).
+		1. Return ? GetWrappedValue(_callerRealm_, _value_).
+	</emu-alg>
+	</emu-clause>
+
+	<emu-clause id="sec-wrappedfunctioncreate" aoid="WrappedFunctionCreate">
+	<h1>WrappedFunctionCreate ( _callerRealm_, _targetFunction_ )</h1>
+	<p>The abstract operation WrappedFunctionCreate takes arguments _callerRealm_ and _targetFunction_. It is used to specify the creation of new wrapped function exotic objects. It performs the following steps when called:</p>
+	<emu-alg>
+		1. Assert: _callerRealm_ is a Realm Record.
+		1. Assert: IsCallable(_targetFunction_) is *true*.
+		1. Let _internalSlotsList_ be the internal slots listed in <emu-xref href="#table-internal-slots-of-wrapped-function-exotic-objects"></emu-xref>, plus [[Prototype]] and [[Extensible]].
+		1. Let _obj_ be ! MakeBasicObject(_internalSlotsList_).
+		1. Set _obj_.[[Prototype]] to %Function.prototype% from _callerRealm_.
+		1. Set _obj_.[[Call]] as described in <emu-xref href="#sec-wrapped-function-exotic-objects-call-thisargument-argumentslist"></emu-xref>.
+		1. Set _obj_.[[Realm]] to _callerRealm_.
+		1. Set _obj_.[[WrappedTargetFunction]] to _targetFunction_.
+		1. Return _obj_.
+	</emu-alg>
+	</emu-clause>
+</emu-clause>
+
 <emu-clause id="sec-realm-objects">
 	<h1>Realm Objects</h1>
+	<emu-clause id="sec-realm-abstracts">
+		<h1>Realm Abstract Operations</h1>
+
+		<emu-clause id="sec-performrealmevaluation" aoid="PerformRealmEvaluation">
+			<h1>PerformRealmEvaluation ( _x_, _evalRealm_ )</h1>
+			<emu-alg>
+				1. Assert: Type(_x_) is String.
+				1. Assert: _evalRealm_ is a Realm Record.
+				1. Let _hostDefined_ be _evalRealm_.[[HostDefined]].
+				1. Let _s_ be ParseScript(_x_, _evalRealm_, _hostDefined_).
+				1. If _s_ is a List of errors, then
+					1. Perform HostReportErrors(_s_).
```

Comment:
I don't find the definition of `HostReportErrors` in this file nor in https://tc39.es/ecma262/

Reply:
Probably from old versions of 262, this was a revert from old versions of this specs anyways. I will fix all that.